text:u'20151115_1200_floor_ceil.csv'
Traceback (most recent call last):
  File "CalculateMC.py", line 67, in <module>
    with open(filename, 'rb') as csvfile:
IOError: [Errno 2] No such file or directory: 'DATA\\preProcessed\\20151116_225244.txt'
